_data/blogs.yml

```diff
@@ -45,3 +45,9 @@
   blog_img: adarsh-blog2.png
   article: What I learned after participating in GCI
   article_link: https://idealadarsh.wordpress.com/2016/12/08/what-i-learned-after-participating-in-google-code-in/
+
+- name: Tinh Dao
+  blog: http://dcongtinh.blogspot.com/
+  blog_img:
+  article: GCI 2016
+  article_link: http://dcongtinh.blogspot.com/2016/12/google-code-in-2016-google-code-in.html
```

Comment:
Blog has already been added in PR #316
Pls, sync the repo, make a new branch from your forked one and then send a PR for adding the mentor. Thanks !